### PR TITLE
- adds go 1.20

### PR DESCRIPTION
**Pull request by baywet**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95835)

### CODE REVIEW

**Comment by baywet** [comment]
@denelon now that the validation is completed, can somebody review and merge it please?

**Review by vedantmgoyal9 — DISMISSED** [review]

File: manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: GoLang.Go.1.20
+PackageVersion: "1.20"
```
Comment (Trenly) [inline manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.installer.yaml:5]:
```suggestion
PackageVersion: "1.20.0"
```

File: manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: GoLang.Go.1.20
+PackageVersion: "1.20"
```
Comment (Trenly) [inline manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "1.20.0"
```

File: manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: GoLang.Go.1.20
+PackageVersion: "1.20"
```
Comment (Trenly) [inline manifests/g/GoLang/Go/1/20/1.20/GoLang.Go.1.20.yaml:5]:
```suggestion
PackageVersion: "1.20.0"
```

**Review by Trenly — DISMISSED** [review]
Looks like the 3-field version is used in control panel. I'm going to approve this since it *Shouldnt* cause any issues, as all versions are assumed to have infinite trailing zeros, so `1.20` and `1.20.0` are functionally equivalent, but for future releases the 3-field version (e.g `1.20.1`) will probably be required, and may require this manifest to be updated later

**Comment by baywet** [comment]
@denelon @hackean-msft the validation bot seems to be on an infinite loop here, is there any way you can help unblock it please?

**Comment by denelon** [comment]
@baywet we're working through some issues with the bots. This one passed validation, but there are open discussion items. You can either mark them as Resolved and the PR will merge, or you can accept the suggestions by @Trenly and then the validation will run again, and close.

**Comment by baywet** [comment]
@denelon, thanks  for the help. I missed the conversation must be resolved, maybe updating the bot to mention that aspect in the future would be helpful (it's noisy so it draws attention out of the PR status panel)

**Review by Trenly — APPROVED** [review]

**Pull request merged**